packages/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -201,6 +213,17 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
           markersController.removeMarkers((List<Object>) markerIdsToRemove);
           break;
         }
+      case "polylines#update":
+        {
+          Object polylinesToAdd = call.argument("polylinesToAdd");
+          polylinesController.addPolylines((List<Object>) polylinesToAdd);
+          Object polylinesToChange = call.argument("polylinesToChange");
+          polylinesController.changePolylines((List<Object>) polylinesToChange);
+          Object polylineIdsToRemove = call.argument("polylineIdsToRemove");
+          polylinesController.removePolylines((List<Object>) polylineIdsToRemove);
```

Comment:
Add `result.success(null)`. I forgot to do it for markers, if you rebase to the tip of master you should see that change :)